src/app/community/create/create.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-card class="communityCreationForm">
+  <mat-card-title>Community Creation Form</mat-card-title>
```

Comment:
I think it could just be "Community Creation" for the title

Reply:
fixed.

Reply:
fixed